### PR TITLE
[JENKINS-37343] added DataBoundConstructor and Symbol for compatibility with DSLs

### PR DESCRIPTION
**Pull request by daspilker**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -145,6 +145,11 @@
     </dependency>
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>structs</artifactId>
```
Comment (simschla) [inline pom.xml]: for which part is this dependency needed? Symbol?

Comment (daspilker) [inline pom.xml]: Yes, it's needed for `@Symbol`. See the jenkinsci-dev mailing list for details: https://groups.google.com/forum/#!msg/jenkinsci-dev/6ijgyOFvZbg/MDIOUKm0BAAJ

**Comment by daspilker** [comment]
@simschla is there anything else I can do to get this merged?

**Comment by Oppodelldog** [comment]
I guess this PR will make the ci-game callable from pipeline steps as well. great!

**Comment by michbeck100** [comment]
I tried to call the ci-game from a pipeline step, but i couldn't get it to work. Do you have an example?

**Comment by aadis** [comment]
Likewise, what's the syntax for calling from Jenkinsfile?

**Comment by arthurplma** [comment]
Hi All,
Not clear to me how to use it in my groovy script. Is this the only way:
```
//CI game
configure { project ->
    project / publishers << 'hudson.plugins.cigame.GamePublisher' {}
}
```

**Pull request merged**